content/aips/942c023868ca-AIP-STKAAVE_RETRIEVAL.md

```diff
@@ -0,0 +1,77 @@
+---
+
+Title: [AIP] Whitelist Balancer’s Liquidity Mining Claim
```

Comment:
You can remove the `[AIP]`. Have the title as just `Whitelist Balancer’s Liquidity Mining Claim`

Reply:
Also `title` instead of `Title`